index.bs

```diff
@@ -3402,6 +3390,19 @@ Maturity Stages on the Recommendation Track</h4>
 			The W3C Royalty-Free IPR licenses
 			granted under the W3C Patent Policy [[PATENT-POLICY]]
 			apply to [=W3C Recommendations=].
+
+			<p id="expandable-rec">
+			After its initial publication,
+			a [=W3C Recommendation=] <em class=rfc2119>may</em> be [[#revising-rec|revised]]
+			to address [=editorial change|editorial=] or [=substantive change|substantive=] issues
+			that are discovered later.
+			However, <a href=#class-4>new features</a> can only be added
+			if the document already identifies itself
+			as intending to <dfn>allow new features</dfn>.
+			Such an allowance cannot be added
+			to a [=technical report=] previously published as a [=Recommendation=]
+			that did not allow such changes.
```

Comment:
This is too broad a restriction - it _is_ permitted if going back to CR and then to Rec, but reading this, it looks like that route is also blocked.

Reply:
It is indeed blocked, and that's not new (this text is moved, not introduced).
* Prior to Process 2020, if a document has made it to REC, you cannot add new features to it, even by going back to CR or WD. You must start a new one. See https://www.w3.org/2019/Process-20190301/#revised-rec:
  > To make changes which introduce a new feature or features, W3C must follow the full process of advancing a technical report to Recommendation beginning with a new First Public Working Draft.
* From Process 2020 onward:
  * the same is true of specs that do not carry the "allows new feature" text. See https://www.w3.org/2020/Process-20200915/#revised-rec-features:
    > To make changes which introduce a new feature to a Recommendation that was not approved for accepting new features, W3C must create a new technical report, following the full process of advancing a technical report to Recommendation beginning with a new First Public Working Draft.
  * specs that do carry the "allows new feature text" can gain new features, but that text must be added prior to the spec reaching REC *for the first time*. See https://www.w3.org/2020/Process-20200915/#transition-pr:
    > A Proposed Recommendation may identify itself as intending to allow new features (class 4 changes) after its initial publication as a Recommendation, as described in § 6.2.11.4 Revising a Recommendation: New Features. **Such an allowance cannot be added to a technical report previously published as a Recommendation that did not allow such changes**. 

The goal is to provide a guarantee to external users: as has always been the case, if you see a REC that does not carry any mention about allowing new features (and no pre-2020 REC has that), you can depend on subsequent publications of the same REC not adding new features. Whether this is a good idea is something we can debate, but that's orthogonal to this Pull Request, so if you want a discussion on this topic, I'd request filing a new issue.

Reply:
Oh, right, I think the confusion here is that you (and the Process) would consider a new _version_ of an existing REC to be a _new REC_ (which is technically correct), whereas a lot of readers would think of it as the same REC with some additional features. Perhaps it would be helpful to add a note here that features can be added to existing RECs that do not "allow new features" by generating a new version.

Reply:
I cannot apparently un-resolve this comment thread, but that's what I'd like to do right now, given my previous comment.

Reply:
> I cannot apparently un-resolve this comment thread, but that's what I'd like to do right now, given my previous comment

Done.

> Perhaps it would be helpful to add a note here that features can be added to existing RECs that do not "allow new features" by generating a new version.

That statement exists as the last paragraph/sentence of https://www.w3.org/policies/process/#revised-rec-features. I am hesitant to make the Process longer by having various parts repeat what other parts already say. Short notes in the right context pointing to longer requirements elsewhere can helpful on occasion, but here I'm not sure how to have something that's useful enough while keeping things short enough.

Reply:
I think a useful note here would be a reminder that new versions (with new version numbers) of existing Recs count as new technical reports. That's not present in the linked section on new features either.